test/e2e/azure_test.go

```diff
@@ -597,4 +597,55 @@ var _ = Describe("Workload cluster creation", func() {
 			})
 		})
 	})
+
+	Context("Creating clusters using clusterclass [REQUIRED]", func() {
```

Comment:
Because the canonical value prop of ClusterClass is > 1 clusters from a common class, should we do that here as well?

Reply:
I thought about this but didn't want to get too much in one pr. The test itself creates a `topology` first before creating the cluster. I'll create a followup pr that creates a more clusters from the topology.  wdyt?

Reply:
Should work. I do wonder about these adding additional test overhead to the default job (identified by the `[REQUIRED]` substring). @CecileRobertMichon any thoughts on whether or not we should run ClusterClass tests by default?

Reply:
Sorry for the late response. Since this is an experimental feature and support in CAPZ is brand new, let's add it as optional? Then, once we've had good mileage on it, we can move it to required and/or migrate existing tests to use clusterclass, WDYT?

Reply:
Moved it to Optional.